### DISCUSSION TITLE
Rename webapp

### DISCUSSION THREAD

**Discussion by shivarm** [Hyperfoil/Horreum#730] [General]
https://github.com/Hyperfoil/Horreum/blob/5d96c79b45d74035227f0c6fa149a66c641ba1d4/webapp/package.json#L2

we should change the name of our ui webapp project and we should keep  `Horreum-UI` or ..... something ?

@jesperpedersen @johnaohara WDYT?

**Comment by jesperpedersen** [comment]
I'm ok with horreum-web
